converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/ExceptionStorage/IExceptionStore.cs

```diff
@@ -29,8 +29,9 @@ public interface IExceptionStore
         /// </summary>
         /// <param name="changeFeedEntry">ChangeFeedEntry that threw exception</param>
         /// <param name="retryNum">Number of times the entry has been tried</param>
+        /// <param name="nextDelayTimeSeconds">Amount of time to wait before retrying</param>
         /// <param name="exceptionToStore">The exception that was thrown and needs to be stored</param>
         /// <param name="cancellationToken">Cancellation token.</param>
-        Task WriteRetryableExceptionAsync(ChangeFeedEntry changeFeedEntry, int retryNum, Exception exceptionToStore, CancellationToken cancellationToken = default);
+        Task WriteRetryableExceptionAsync(ChangeFeedEntry changeFeedEntry, int retryNum, double nextDelayTimeSeconds, Exception exceptionToStore, CancellationToken cancellationToken = default);
```

Comment:
A lot of APIs in .NET that have a delay or wait typically either use milliseconds (as `int`) or a `TimeSpan`. Eg. [`Task.Wait`](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.wait?view=net-5.0)